service/mock/k8s/admin.conf

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMvakNDQWVhZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJeU1EVXlOekUwTURNMU5sb1hEVE15TURVeU5ERTBNRE0xTmxvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTWFVCmovbFYyeWhaMjdIUm9xMFJSVitOa052NUhUT1dXVW1ocS9FbkVmKzRlYlJVcXdBZHIyQUZjWEVhZ2dxYWpaU1gKdWIvWGhmUVM1MVkvQnhPMStlRXBWWkZtVkFSL3Qra0JwRGJNTHhwRVdMUFp1cXpFV2xTYWxYRzNERm9FeGxwWQpVY0FUMVk2cVBNb2d4M2NySFFwYWFKcUIzVVNDa0FWZlpwcnZGd05oZ2l4ZHNXS096ZWNVQ2NudFVaVWJMY21wClBIREg1QUJXanFWdVo2bDJzU2ZhYm9VZTdNMjc4TkZkdHJnSzJ6MDV0UGtnVjlpblJoWi8wK1BnemlsZEZ3dkQKOVFXK0pIVEQ5dFJTQWpvakx6RlQrSDBOdjZMVzFuWWk4UXNPM01EMXJoUGFYTDZVYWtheHlseGJtWUp0bWN3KwpZcFdVWEZKNG0vMzQveG03dlFNQ0F3RUFBYU5aTUZjd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0hRWURWUjBPQkJZRUZOTXgrMlEvODJkV05pazM2K3JBRjhFUXd5cFZNQlVHQTFVZEVRUU8KTUF5Q0NtdDFZbVZ5Ym1WMFpYTXdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUJBQTJsWHFsTDk0eThnSHZlMUVQUwo0a2tSVEJ4a1N3NlBFbm1ybGNCY08xN3BJVzE4QXFndnZSKzVaMEVnOFZ6TDZXdDJnN01yVVI2QXJEV0RMTWlUClNiaUZMdzJMQmZvSktkUHZhZXZXcFptZ3d0TjY5V3ByQmxiamI5M0hFNS9KMGtNTjRhaTFjMkdFNWdUL0tadkEKWTBwbGlhd09MMDlHQkJmaERWcWh5UFU4aU5FRS95MDJKenlkbGlqc1VaSkx3VXB6WGV4TUFaTVhYK1h1OVRLYwpQTkUvZy9tSFhEZDVOcFZ0ZVpiQm0vd2pTN3lkZlNnbFhMMDgzcHN0K29BRVEyRkN3Q05BbStCb2wrMWVUcTRuCnZFTVkwYmR1SWtmdk9hemp6SWlvZkR1TjkxY0tJWGJQcXdjc0hybzdoakVUVUZLRXFtaDZ1TVp5SWZ4aVZVOHIKNEs4PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
+    server: https://10.247.66.177:6443
```

Comment:
IP can be changed since this is a mock request

Reply:
changed to 127.0.0.1:36443